os/arch/arm/src/stm32l4/stm32l4_ospi.c

```diff
@@ -1920,9 +1920,7 @@ FAR struct mtd_dev_s *up_flashinitialize(void)
     priv->bwrite = this_bwrite;
     priv->read = this_read;
     priv->ioctl = this_ioctl;
-#ifdef CONFIG_MTD_REGISTRATION
-    priv->mtd.name = "stm32l4_flash";
-#endif
+
```

Comment:
Why is a single extra-line addition change made here?
I think this might be due to some re-base confusion.
Is this related to the issue?